src/markdown-pages/docs/writing-own-tests/setting-up-galasa-project.md

```diff
@@ -3,369 +3,191 @@ path: "/docs/writing-own-tests/setting-up-galasa-project"
+
+In the following example we are going to use the Galasa CLI to build a hierarchy of projects, where the parent project _dev.galasa.example.banking_ contains the following modules: 
+- Two test sub-projects or modules called _dev.galasa.example.banking.payee_ and _dev.galasa.example.banking.account_ 
+- A mandatory OBR sub-project called _dev.galasa.example.banking.obr_.
 
-In a temporary directory - and you can do this with whatever tools you choose, no need to use Eclipse - create the following file structure:
+This setup is designed as an example of how to create a project structure that might ultimately be deployed to a Maven repository, so that your Galasa automation system can find everything it needs to run.
 
+The following example syntax assumes a banking application is being tested. The banking application contains functionality that deals with `payee` and `account` object types, both of which can be manipulated:
+
+```
+galasactl project create \
+--package dev.galasa.example.banking \
+--features payee,account \
+--force \
+--log
 ```
-────com.example.tests.parent
-    │   pom.xml
-    ├───com.example.tests.atests
-    │   │   pom.xml
+
+where <br>
+- ```--package``` is the name of a Java package. This parameter is mandatory. Use the provided example to help you to understand how you might create unique names for your Java test packages. Sections can reflect your company, organisation, the application under test, or any other name-spacing of your choice. The value chosen influences the names of the folders, OSGi bundles, Maven coordinates, and the eventual Java package name in which example tests are created. The name must be all lower-case letters and numbers (`a-z`, `0-9`), with `.` (period) character separators. No part of the name can clash with a Java language reserved word. <br>
+- ```--features``` is a comma-separated list of one or more features of the application that you want to test. This parameter is optional, defaulting to `test`. The entries in this list influence the names of some folders, OSGi bundles, Maven coordinates and Java class names of the generated example tests. In the example provided, the banking application under test has a `payee` section and an `account` section which need testing. Creating subcomponents of your test suite to test these features or components of the application creates a naming structure in which your tests can be organised. These names must be lower-case letters and numbers (`a-z`, `0-9`), with no `.` (period) or special characters. No part of the name can clash with a Java language reserved word. <br>
+- ```--force``` is an optional flag. If the flag is missing, then any file that the Galasa CLI tool tries to create which already exists causes a failure, and the original file is preserved. If this flag is used, then such files are silently over-written. Use this option carefully to avoid some of your files being over-written, resulting in the loss of some of your data.
+
```

Comment:
```
--log -
```
Means to direct the trace/logging generated by the tool directly to the console.

Reply:
added the --log - info